Cannot build for raspbian (Raspberry Pi 3B+)

Following the instructions to build this myself and I'm seeing:

```
cargo install --path .                                                                                                                          master
    Updating registry `https://github.com/rust-lang/crates.io-index`
    Updating registry `https://github.com/rust-lang/crates.io-index`
error: failed to compile `monolith v2.1.2 (file:///home/pi/monolith)`, intermediate artifacts can be found at `/home/pi/monolith/target`

Caused by:
  no matching package named `native-tls-crate` found (required by `reqwest`)
location searched: registry https://github.com/rust-lang/crates.io-index
version required: ^0.2
```

Apologies if I've missed something obvious, this is my first time using `cargo`.

I'm encountering the same issue.

```
bjorn@bjorn-ThinkPad-T420:~/project/monolith$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.10
Release:	18.10
Codename:	cosmic
bjorn@bjorn-ThinkPad-T420:~/project/monolith$ make
    Updating crates.io index
    Updating crates.io index
error: no matching package named `native-tls-crate` found
location searched: registry `https://github.com/rust-lang/crates.io-index`
required by package `reqwest v0.10.0`
    ... which is depended on by `monolith v2.1.2 (/home/bjorn/project/monolith)`
make: *** [Makefile:7: build] Error 101
```

--- a/.github/workflows/cd.yml
+++ b/.github/workflows/cd.yml
@@ -1,21 +1,43 @@
+# CD GitHub Actions workflow for Monolith
+
 name: CD
 
 on:
   release:
-    types: [created]
+    types:
+    - created
 
 jobs:
+
   windows:
     runs-on: windows-latest
-
     steps:
     - run: git config --global core.autocrlf false
     - name: Checkout the repository
-      uses: actions/checkout@master
+      uses: actions/checkout@v2
     - name: Build and install the executable
-      run: cargo install --force --locked --path .
+      run: cargo build --release
     - uses: Shopify/upload-to-release@1.0.0
       with:
         name: monolith.exe
-        path: C:\Users\runneradmin\.cargo\bin\monolith.exe
+        path: target\release\monolith.exe
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+  gnu_linux_armhf:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+    - name: Prepare cross-platform environment
+      run: |
+        rustup target add arm-unknown-linux-gnueabihf
+        git clone https://github.com/raspberrypi/tools.git rpi_tools
+    - name: Build and install the executable
+      run: |
+        export RUSTFLAGS="-C linker=rpi_tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin/arm-linux-gnueabihf-gcc"
+        cargo build --release --target=arm-unknown-linux-gnueabihf
+    - uses: Shopify/upload-to-release@1.0.0
+      with:
+        name: monolith-gnu-linux-armhf
+        path: target/arm-unknown-linux-gnueabihf/release/monolith
         repo-token: ${{ secrets.GITHUB_TOKEN }}

